package.json

```diff
@@ -5,6 +5,11 @@
   "description": "A framework for building native apps using React",
   "license": "MIT",
   "repository": "github:facebook/react-native",
+  "homepage": "https://github.com/facebook/react-native/tree/HEAD",
+  "bugs": {
+    "url": "https://github.com/facebook/react-native/issues"
+  },
+  "keywords": ["react","android","ios","mobile","react-native","cross-platform","app-framework","mobile-development"],
```

Comment:
This is not right place to add `keywords` this `package.json` is not get published to npm. It just adds noise for no value.

Reply:
Yeah. Corrected.